### DISCUSSION TITLE
PHP 7.4 support?

### DISCUSSION THREAD

**Discussion by Luc45** [docker-library/wordpress#811] [General]
Hello,

We have some testing infrastructure build upon these images. Just so that I can plan in advance, I'd like to understand what are the plans for the PHP 7.4 images of WordPress maintained in this repo? How long do you plan to support it?

Thanks!

**Comment by Luc45** [comment]
Pinging @tianon as I'm not sure if notifications are sent when a discussion is started

**Comment by yosifkit** [comment]
The PHP 7.4 based images already pushed to Docker Hub will not be removed, so they will be available indefinitely (although stale). No new images will be pushed since PHP 7.4 is already end of life. PHP 7.4 was removed in https://github.com/docker-library/wordpress/pull/775 (and https://github.com/docker-library/php/pull/1349).

**Reply by Luc45** [reply]
Thank you for the clarification!
